test/integration-legacy/selenium-helpers.js

```diff
@@ -35,7 +35,7 @@ class SeleniumHelper {
         if (remote === 'true'){
             let nameToUse;
             if (ci === 'true'){
-                nameToUse = 'travis ' + buildID + ' : ' + name;
+                nameToUse = 'circleCI ' + buildID + ' : ' + name;
```

Comment:
Or maybe we should just expect this as its own environment variable?

Reply:
Circle has a built in CIRCLECI environment variable set to true and Travis similarly has a TRAVIS environment variable set to true.  Since we're not expecting to use Travis anymore I'm just checking to see if we're using Circle.  If for some reason we're not it'll be set to "unknown".

It didn't seem like it was worth it to check if we were using Travis.

Reply:
Sorry I meant we could pass all of `nameToUse` in one environment variable and move all the logic about what to call it outside of these files.  But I think this is fine as is for now.